Index: Network.cxx
===================================================================
--- Network.cxx
+++ Network.cxx	2020-02-05 19:18:30.042361621 +0100
@@ -107,7 +107,7 @@
     {
       NetworkManager::AccessPoint::Ptr accessPoint = dev->activeAccessPoint();
       int signalStrength = accessPoint.isNull() ? 0 : accessPoint->signalStrength();
-      int x = std::round(signalStrength / 25.0) * 25;
+      int x = qRound(signalStrength / 25.0) * 25;
       pixmap = QIcon::fromTheme(QString("network-wireless-connected-%1").arg(x)).pixmap(size());
 
       if ( !accessPoint.isNull() )
Index: Network.hxx
===================================================================
--- Network.hxx
+++ Network.hxx	2020-02-05 19:19:42.570240666 +0100
@@ -25,6 +25,7 @@
 
 #include <SysTrayItem.hxx>
 #include <NetworkList.hxx>
+#include <QPointer>
 #include <QTimer>
 #include <QPixmap>
 
Index: NetworkList.cxx
===================================================================
--- NetworkList.cxx
+++ NetworkList.cxx	2020-02-05 19:20:53.811228674 +0100
@@ -23,6 +23,7 @@
 #include <QHBoxLayout>
 #include <QCheckBox>
 #include <QToolButton>
+#include <QTimer>
 #include <QDebug>
 
 #include <KLocalizedString>
Index: start_liquidshell
===================================================================
--- start_liquidshell
+++ start_liquidshell	2020-02-05 19:27:07.463986752 +0100
@@ -33,7 +33,12 @@
 Hidden=true
 EOF
 
-/usr/bin/startkde
+if [ -f /usr/bin/startkde ]
+then
+  /usr/bin/startkde
+else
+  /usr/bin/startplasma-x11
+fi
 
 rm -f $plasmaFileName 
 
