From ec6fe85d8b09343f03e28e38ae54ee5516e1c9cb Mon Sep 17 00:00:00 2001
From: Martin Koller <kollix@aon.at>
Date: Fri, 18 Oct 2019 09:05:37 +0200
Subject: startkde was changed to startplasma-x11

---
 start_liquidshell | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/start_liquidshell b/start_liquidshell
index 66ad1a4..04e258d 100644
--- a/start_liquidshell
+++ b/start_liquidshell
@@ -33,7 +33,12 @@ cat > $plasmaFileName <<EOF
 Hidden=true
 EOF
 
-/usr/bin/startkde
+if [ -f /usr/bin/startkde ]
+then
+  /usr/bin/startkde
+else
+  /usr/bin/startplasma-x11
+fi
 
 rm -f $plasmaFileName 
 
-- 
cgit v1.1

